eng/release/Scripts/InstallAzCopy.ps1

```diff
@@ -0,0 +1,48 @@
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$true)][string] $ToolsDirectory,
+    [Parameter(Mandatory=$false)][string] $TaskVariableName = $null
+)
+
+$ErrorActionPreference = 'Stop'
+Set-StrictMode -Version 2.0
+
+$url = 'https://aka.ms/downloadazcopy-v10-windows'
+$basePath = Join-Path $ToolsDirectory 'azcopy'
+
+$zipPath = Join-Path $basePath 'azcopy.zip'
+$toolDirPath = Join-Path $basePath 'azcopy'
+$azCopyPath = Join-Path $toolDirPath 'azcopy.exe'
+
+if (Test-Path $azCopyPath) {
```

Comment:
Is the file persistent from build to build? Will this ever pick up updated versions of azcopy?

Reply:
It won't persist from build to build because of how the agents are acquired. This is more of a "if someone calls this again in the same job, don't do extra work." type of thing.